src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/ProcessDeclarationsTransactions.sql

```diff
@@ -18,6 +32,10 @@ INSERT INTO [employer_financial].LevyDeclarationTopup
 		[employer_financial].[GetLevyDeclarationAndTopUp] x
 	where
 		x.LevyDueYTD is not null AND x.LastSubmission = 1 AND x.AccountId = @AccountId AND x.EmpRef = @EmpRef
+	AND @EarliestPositiveLevyDecYear IS NOT NULL
```

Comment:
I think the earliest possible levy declaration filter should only apply if it's the first time levy declarations for the PAYE scheme are added to the account.

Reply:
fixed in later commits